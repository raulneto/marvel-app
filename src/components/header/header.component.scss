header {
    background-color: #dd2323b8;
    position: absolute;
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-content: space-evenly;
    align-items: center;
    height: 150px;
}

.search {
    input {
        color: #666;
        border: 0 none;
        border-radius: 4px;
        height: 34px;
        width: 380px;
        box-shadow: 2px 2px 0 rgba(0,0,0,.3);
        padding: 5px 13px;
        box-sizing: border-box;
        font: italic 14px Arial, sans-serif;
    }
}