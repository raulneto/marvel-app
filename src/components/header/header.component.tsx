import * as React from "react";
import './header.component.scss';
import logo from '../../assets/images/marvel_logo.png';
import { SearchComponent } from '../search/search.component'

export const HeaderComponent: React.FC = () =>  {
    return (
        <header>
            <div className="logotipo"><img src={logo} /></div>
            <div className="search">
                <SearchComponent />
            </div>
        </header>
    )
}