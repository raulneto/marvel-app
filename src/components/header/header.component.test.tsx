import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { HeaderComponent } from './header.component';
import { QueryClient, QueryClientProvider } from 'react-query';

const queryClient = new QueryClient();

test('HeaderComponent render', () => {
	const component = shallow(<HeaderComponent />);
	expect(toJson(component)).toMatchSnapshot();
});
