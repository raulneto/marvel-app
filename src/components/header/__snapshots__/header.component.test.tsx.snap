// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent render 1`] = `
<header>
  <div
    className="logotipo"
  >
    <img
      alt="Marvel App"
      src="marvel_logo.png"
    />
  </div>
  <div
    className="search"
  >
    <withRouter(Search) />
  </div>
</header>
`;
